install-dependencies.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+
+debian_packages=(
+    libldap2-dev
+    libsasl2-dev
+    libssl-dev
+    python3-dateutil
+    python3-memcache
+    python3-dev
+)
+
+redhat_packages=(
+    python3-memcache
+    python3-dateutil
+    python3-devel
+    openldap-devel
+)
+
+pip_modules=(
+    matrix_client
```

Comment:
pip install relies in https://github.com/psaavedra/matrix-bot/blob/main/setup.py. It is important to keep the track of the installed versions like the setup.py does: https://github.com/psaavedra/matrix-bot/blob/main/setup.py#L59

I think you could do a similar thing with something like this:

```
pip_modules=(
    matrix-client==0.0.6
    ...
)
```

But still I dont like the idea of to have the dependencies listed in two files at the same time.

Reply:
I agree, it's better to keep the dependencies in one place. I was not aware of this `setup.py` file.

Mainly the problem I was trying to solve is that whenever I run the matrix-bot from a new host, I spend a few minutes solving dependencies until the bot runs successfully. So I would like to have a script that I can run and got all the dependecies solved.

So it occurs to me to keep this install-dependencies.sh script for required packages only (not Python modules), and then allow `setup.py` to be executed directly to install de Python modules. WDYT?

Reply:
Sounds reasonable.